llvm/lib/Transforms/Coroutines/CoroFrame.cpp

```diff
@@ -52,6 +53,16 @@ extern cl::opt<bool> UseNewDbgInfoFormat;
 
 enum { SmallVectorThreshold = 32 };
 
+static std::string getBasicBlockLabel(const BasicBlock *BB) {
+  if (BB->hasName())
+    return BB->getName().str();
```

Comment:
This function could have returned a non-owning `StringRef`. This `.str()` here creates a copy of the string.

Reply:
If we just use BB->printAsOperand(dbgs(), false) we can remove this need to go to a string. But that would result in a % being printed before the BB label. I find that a bit confusing because BB labels are not otherwise printed with a preceding %.

Reply:
For example, when print remat right now I realized it is using BB->printAsOperand(dbgs(), false) but that results in output like

Entry (%1122) :   %1182 = call ...

So the % is a bit confusing there.

Reply:
I prefer it to print

Entry (1122) :   %1182 = call ...